libbpf-tools/tcpretrans.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Anton Protopopov
+// Copyright (c) 2021 Red Hat, Inc.
+#ifndef __TCPRETRANS_H
+#define __TCPRETRANS_H
+
+#define MAX_ENTRIES 8192
```

Comment:
This could be defined in .bpf.c.

Reply:
MAX_ENTRIES is also used in tcpretrans.c